x-pack/plugin/fleet/src/javaRestTest/java/org/elasticsearch/xpack/fleet/FleetDataStreamIT.java

```diff
@@ -48,6 +48,11 @@ protected Settings restAdminSettings() {
             .build();
     }
 
+    @Override
+    protected boolean preserveSecurityIndicesUponCompletion() {
+        return true;
```

Comment:
Is this for performance reasons?

Reply:
Yes, in a way. The fleet IT test do not interact with `.security` index directly, so I started adjusting them and decided to keep `.security` index also for simplicity of assertions. But then decided to do that in a followup.